rllib/agents/ppo/ppo_torch_policy.py

```diff
@@ -2,7 +2,7 @@
 from typing import Dict, List, Type, Union
 
 import ray
-from ray.rllib.agents.ppo.ppo_tf_policy import setup_config
+from ray.rllib.agents.ppo.ppo_tf_policy import validate_config
```

Comment:
See above: Should this be a class method, instead of a utility function?

Reply:
oh, this is a util function because we seem to be sharing the ppo validate_config with a few our agents.
if it's a class method, we will be importing the entire DynamicPPOTFPolicy class, and only use the validate_config call, which may look weird.

Reply:
Makes sense!